ui/desktop/app/components/settings-card/client-agent/index.hbs

```diff
@@ -0,0 +1,72 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if this.isClientAgentActive}}
+  <SettingsCard
+    @header={{t 'settings.client-agent.title'}}
+    @icon='network-alt'
```

Comment:
based on figma, should it be [this](https://helios.hashicorp.design/icons/library?searchQuery=icon%3Amigrate&selectedGroupType=alphabetical&selectedIconSize=24)?

Reply:
I actually changed it because I thought it was a better icon to represent something "networky" since the arrows didn't mean much I thought, I forgot to check in with design though